index.html

```diff
@@ -869,7 +890,27 @@ <h3>Script Event</h3>
 
         <section>
           <h4>Text</h4>
-         <p>The <dfn>Text</dfn> object contains text content typically in a single language. This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p>The <dfn>Text</dfn> object contains text content typically in a single language.
+            This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p><a>Text</a> is defined as <dfn>Original</dfn> if it is any of:
+            <ol>
+              <li>the same language as the dialogue that it represents in the original programme audio;</li>
+              <li>a transcription of text visible in the programme video, in the same language as that text;</li>
+              <li>an untranslated representative of non-dialogue sound;</li>
+              <li>an untranslated description of the scene in the programme video.</li>
+            </ol>
+          </p>
+          <aside class="note">The language of an <a>Original</a> <a>Text</a> object
+            can be different to the document's <a>Default Language</a>.</aside>
+          <p><a>Text</a> is defined as <dfn>Translation</dfn> if it is
+            a representation of an <a>Original</a> <a>Text</a> object in a different language.</p>
+          <p><a>Text</a> can be <em>identified</em> as being <a>Original</a> or <a>Translation</a>
+            by inspecting its language and its <a>Text Language Source</a> together,
+            according to the semantics defined in <a>Text Language Source</a>.</p>
+          <p>The source language of <a>Translation</a> <a>Text</a> objects and, where applicable,
+            <a>Original</a> <a>Text</a> objects
+            is indicated using the <a>Text Language Source</a> property.</p>
```

Comment:
It feels to me that the 2 sentences are saying the same thing. Maybe keep only one?

Reply:
I don't think they are saying the same thing. The first is about identifying Original vs Translation, the second is about what the source language was.